### PR TITLE
Create a page dedicated to contributors

### LINKED ISSUE #1414: Page dedicated to contributors
It would be nice to have a complete list of contributors to the project for people to browse.  I threw together a quick (and admittedly not great) Elixir script to figure that out for us: https://gist.github.com/doomspork/2edf3e64ae229beacccb694cf9191969

There's handful (< 6) users who've deleted their accounts and were pushed into the `"missing_author"` group.  I'll manually work through those.  

This is the generated list of contributors and first commits:

doomspork - 2015-05-31
GabKlein - 2015-06-08
leifg - 2015-08-19
Tica2 - 2015-10-09
jinyeow - 2015-10-30
asarode - 2015-11-28
erickgnavar - 2015-11-30
3100 - 2015-12-02
xxyxyx - 2015-12-04
ZuraGuerra - 2015-12-09
tiagoamaro - 2015-12-13
Techbrunch - 2015-12-14
upamune - 2015-12-16
kianmeng - 2015-12-19
morty - 2015-12-22
milmazz - 2015-12-28
cizixs - 2016-01-13
rsj217 - 2016-01-18
pjhampt- - 2016-01-22
jesusangelm - 2016-01-23
zhyu - 2016-01-25
iamrahulroy - 2016-01-31
tiagobarreto - 2016-02-07
roadev - 2016-02-07
j-eslee85 - 2016-02-18
brain-geek - 2016-02-19
vnnoder - 2016-02-19
pragmaticivan - 2016-02-23
michalvalasek - 2016-02-23
rodrigoddalmeida - 2016-02-27
brunoasantos - 2016-02-27
psantos10 - 2016-02-29
m0a - 2016-03-01
timlew - 2016-03-02
inchingforward - 2016-03-03
jessejanders- - 2016-03-05
philss - 2016-03-10
ruan-brandao - 2016-03-10
derick - 2016-03-13
Slotos - 2016-03-14
Denys-Soares - 2016-03-15
Diog-eves - 2016-03-17
joeltrew - 2016-03-20
beeetooo - 2016-03-21
jadercorrea - 2016-03-21
victorlcampos - 2016-03-22
gaynetdinov - 2016-03-23
tchoutri - 2016-03-23
julienXX - 2016-03-24
Kosmas - 2016-03-25
ajhager - 2016-03-26
davidvanegas7 - 2016-03-27
renatosuero - 2016-03-28
nscycl-e - 2016-03-28
andreapav-i - 2016-03-30
apeacox - 2016-03-30
ilasorsa - 2016-04-03
kerlak - 2016-04-04
RTJ - 2016-04-05
veverkap - 2016-04-11
alkaha - 2016-04-12
kdebowski - 2016-04-13
aquarhead - 2016-04-14
jakefolio - 2016-04-14
burden - 2016-04-16
Koziolek - 2016-04-21
cefigueiredo - 2016-04-21
rezaprima - 2016-04-23
Devalo - 2016-04-27
setzer777 - 2016-04-27
jc00ke - 2016-05-01
devleoper - 2016-05-02
jeanmatheussouto - 2016-05-08
Dalg-a - 2016-05-10
ybycode - 2016-05-10
henry-hz - 2016-05-10
malkoG - 2016-05-10
pcewing - 2016-05-20
fab--f - 2016-05-20
aleandros - 2016-05-22
kyledecot - 2016-05-23
yyyc514 - 2016-05-28
edshadi - 2016-05-30
HirofumiTamori - 2016-05-31
Zahidul-Islam - 2016-06-01
riseshia - 2016-06-27
hubertlepicki - 2016-06-30
Nagasaki45 - 2016-06-30
ybod - 2016-07-02
markgeek - 2016-07-04
mikaoelitiana - 2016-07-04
GregMefford - 2016-07-08
MuriloDalRi - 2016-07-10
originerd - 2016-07-16
chareice - 2016-07-16
ybur-yug - 2016-07-18
merkata - 2016-07-19
jswny - 2016-07-19
christiansakai - 2016-07-20
yepesasecas - 2016-07-20
gemantzu - 2016-07-26
mercul3s - 2016-07-26
alexwbuschle - 2016-07-26
AdamT - 2016-07-28
lucidstack - 2016-07-29
benjic - 2016-08-01
gmile - 2016-08-01
araslanov-e - 2016-08-03
scouten - 2016-08-05
eze1981 - 2016-08-07
arelangi - 2016-08-09
shibing - 2016-08-16
hasanozgan - 2016-08-20
firec0der - 2016-08-22
j-ahoffline - 2016-08-29
PatNowak - 2016-08-30
marocchino - 2016-08-31
ipmsteven - 2016-09-09
fleveque - 2016-09-12
marvin-min - 2016-09-12
holsky - 2016-09-21
AgIt0 - 2016-09-25
luzeduardo - 2016-10-06
brunogirin - 2016-10-08
anderk-zen - 2016-10-13
Lechindianer - 2016-10-18
et - 2016-10-19
zaid - 2016-10-20
mijailr - 2016-10-22
basavind - 2016-10-24
v-kolesnikov - 2016-10-27
dmvt - 2016-11-02
revdan - 2016-11-05
blogscot - 2016-11-08
kitlangt- - 2016-11-10
princemaple - 2016-11-11
PericlesTheo - 2016-11-14
rafalbig - 2016-11-15
ShallmentMo - 2016-11-21
j-asporto - 2016-11-21
jumichot - 2016-11-21
techgaun - 2016-11-22
halostatue - 2016-11-26
jeramyRR - 2016-11-28
ayayalar - 2016-11-29
csytan - 2016-12-05
andresilveirah - 2016-12-06
adampash - 2016-12-09
LostKobrakai - 2016-12-10
mrefish - 2016-12-10
saneery - 2016-12-14
prio101 - 2016-12-15
stosorio - 2016-12-15
jmitchell - 2016-12-18
capaci - 2016-12-22
namjae - 2017-01-05
BryanJBryce - 2017-01-10
Jas-YCHuang - 2017-01-10
lk-geimfari - 2017-01-13
moxley - 2017-01-16
Raincal - 2017-01-16
utcnow - 2017-01-20
kiennt - 2017-01-23
Suckzoo - 2017-01-23
masnun - 2017-02-16
code-shoily - 2017-02-16
xinz - 2017-02-19
WDoan - 2017-02-27
aseigo - 2017-02-27
pedrosnk - 2017-03-03
fdbeirao - 2017-03-05
pr-dzyn - 2017-03-08
norbajunior - 2017-03-12
slavo2 - 2017-03-20
nurugger07 - 2017-03-25
edwinallenz - 2017-03-27
hanmd82 - 2017-03-27
toothfairy - 2017-03-29
blueringlab - 2017-03-30
rrmartins - 2017-04-02
vicdda - 2017-04-03
rodrigodealer - 2017-04-04
meox - 2017-04-08
recomap - 2017-04-08
nanl-g - 2017-04-11
Cifer-Y - 2017-04-11
rinosamakanata - 2017-04-12
mor-tt - 2017-04-14
Shkrt - 2017-05-09
eksperimental - 2017-05-14
feynmanliang - 2017-05-14
jfornoff - 2017-05-24
nhooyr - 2017-05-30
joeunha - 2017-06-13
ruslanshakirov - 2017-06-14
wellls - 2017-06-21
djKooks - 2017-07-10
bguiz - 2017-07-30
caio - 2017-07-30
RichMorin - 2017-08-01
zhiephie - 2017-08-02
bbuck - 2017-08-11
harfangk - 2017-08-13
dyancarlos - 2017-08-13
rodrigopinto - 2017-08-27
chriseyre2000 - 2017-08-27
trofivan - 2017-08-28
Ljzn - 2017-09-11
romenigld - 2017-09-11
picandocodigo - 2017-09-13
dev-estes - 2017-09-13
elpikel - 2017-09-22
martosaur - 2017-09-23
hlship - 2017-09-23
zackchandler - 2017-10-01
iboss-ptk - 2017-10-01
ismailunal - 2017-10-02
gavlak - 2017-10-02
akkatracker - 2017-10-02
crab-ature - 2017-10-05
kalarani - 2017-10-09
Daqu - 2017-10-10
lukkiddd - 2017-10-13
zkan - 2017-10-13
phoomparin - 2017-10-13
zixsma - 2017-10-13
gomaster-me - 2017-10-17
glazzari - 2017-10-19
theazharul - 2017-10-20
araujoigor - 2017-10-20
jas-n-stop10 - 2017-10-21
stfnsr - 2017-10-21
clairt- - 2017-10-24
dukex - 2017-10-25
squaresurf - 2017-10-26
iguchi1124 - 2017-11-06
SpyR1014 - 2017-11-08
blissjaspis - 2017-11-09
ramytamer - 2017-11-10
flav-o123 - 2017-11-14
lithiumpie - 2017-11-30
eam-taaffe - 2017-12-04
doroshenkoss - 2017-12-09
jhsbeat - 2017-12-26
pmutua - 2018-01-05
gimK-do - 2018-01-07
brunoliveira8 - 2018-01-09
mbirman - 2018-01-09
Vuta - 2018-01-09
jrichocean - 2018-01-09
andersao - 2018-01-19
kalpabarbosa - 2018-01-21
yasinyaman - 2018-02-09
MertcanGokgoz - 2018-02-10
nathanl - 2018-02-11
thiamsantos - 2018-02-18
mememori - 2018-02-20
ueneid - 2018-03-13
mikeover - 2018-03-15
chadoh - 2018-03-19
wilbertliu - 2018-03-23
phips4 - 2018-03-23
jordan-brough - 2018-03-30
justanotherdot - 2018-03-30
sgnl - 2018-04-05
BEaStia - 2018-04-10
snamiki1212 - 2018-04-16
cadebward - 2018-04-21
chuckhousley - 2018-04-21
inodaf - 2018-04-24

### PR DESCRIPTION
**Pull request by doomspork**
Closes #1414 

This still needs some CSS lovin'

### CODE REVIEW

File: contributors.html [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+      <li>2016-03-21 - <a href="https://github.com/jadercorrea">jadercorrea</a></li>
+      <li>2016-03-22 - <a href="https://github.com/victorlcampos">victorlcampos</a></li>
+      <li>2016-03-23 - <a href="https://github.com/gaynetdinov">gaynetdinov</a></li>
+      <li>2016-03-23 - <a href="https://github.com/tchoutri">tchoutri</a></li>
+      <li>2016-03-24 - <a href="https://github.com/julienXX">julienXX</a></li>
+      <li>2016-03-25 - <a href="https://github.com/Kosmas">Kosmas</a></li>
+      <li>2016-03-26 - <a href="https://github.com/ajhager">ajhager</a></li>
+      <li>2016-03-27 - <a href="https://github.com/davidvanegas7">davidvanegas7</a></li>
+      <li>2016-03-28 - <a href="https://github.com/renatosuero">renatosuero</a></li>
+      <li>2016-03-28 - <a href="https://github.com/nscycl-e">nscyclone</a></li>
```
Comment (nscyclone) [inline contributors.html]: Something went wrong with the Github links. For example this and the next one seem to be incorrect.

**Comment by doomspork** [comment]
@nscyclone that should be fixed, I just hadn't pushed yet 😀 

Also removed the social links.  We'll leave it to GitHub for now.

**Comment by doomspork** [comment]
@eksperimental 

image

Do you think we should include a role (ie Japanese Translator, Russian Translator)?  Would that add value?

**Comment by eksperimental** [comment]
I would create groups and list people. It may help finding people, and repeat names in different groups if necessary.. anf yeah.. the role , translator, reviewer, developer, etc

**Comment by eksperimental** [comment]
It will definitely add value

**Comment by doomspork** [comment]
@eksperimental I'll take a stab at this today and ping you when I have some changes for you to look at.  Thanks again for all of your help, it's much appreciated 😁

**Comment by doomspork** [comment]
How's this @eksperimental?  

image

**Comment by eksperimental** [comment]
Looks great, but creating a subgroup for every language would look much neater IMO

**Comment by doomspork** [comment]
@eksperimental thoughts on listing lessons contributed by authors?

image

**Comment by eksperimental** [comment]
Sorted by first contribution, but omitting the date as you did looks great

**Comment by doomspork** [comment]
image

**Comment by eksperimental** [comment]
I think a heading for translators, and a subheading for each language is better

**Comment by eksperimental** [comment]
And languages sorted alphabetically

**Comment by doomspork** [comment]
CSS is most definitely not my forte so I don't want to spend too much time on that.  Let's get the structure where we want and I'll tag @burden to make 'er look good.

image

**Comment by doomspork** [comment]
@eksperimental updated and sorted languages 👍

File: _data/contributors.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  - language: Tamil
+    users:
+      - kalarani
+
+  - language: Thai
+    users:
+      - iboss-ptk
+
+  - language: Traditional Chinese
```
[the remark this answers is not in the document]
Comment (doomspork) [reply _data/contributors.yml]: Should it be listed as "Chinese, Traditional" ?

**Comment by doomspork** [comment]
@eksperimental I'm thinking for now we just have this one in English for the time being.  I anticipate folks will be able to get the gist of it without needing to copy-pasta it throughout the project and require additional work out of translators.

If someone wants to improve this in the future I'm all for it 😀

**Pull request merged**